### PR TITLE
Feat: Add new notes, moved notes to recycle bin and delete note

### PR DESCRIPTION
**Pull request by abhijeetekad**
Added new notes, remove notes and delete notes from recycle bin. 
chrome-capture (15)

### CODE REVIEW

File: src/Components/Sidebar/Sidebar.jsx [hunk 1]
```diff
@@ -11,13 +11,13 @@ function Sidebar() {
         <NavLink className="link" to="/main-page">
```
Comment (shilpe26) [inline src/Components/Sidebar/Sidebar.jsx:11]: Good that you use `NavLink` but I can't see any property attached to it, if you decide to use it in the future then it's all right👍

**Review by shilpe26 — APPROVED** [review]
▪️The code is pretty clean and properly intended
▪️The naming convention is also relevant to the specific work
Good to merge👍

File: src/Components/NewNote/NewNote.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  border-radius: 5px;
+  background-color: #f2f2f2;
+  padding: 20px;
+  width: auto;
+}
+.form-catageory {
+  display: flex;
+  gap: 1rem;
+  align-items: center;
+  /* justify-content: space-between; */
```
Comment (prathmeshjagtap) [inline src/Components/NewNote/NewNote.css:11]: remove commented  code if not required

**Review by prathmeshjagtap — APPROVED** [review]
code is Well structured and you have written a very clean code

File: src/Components/Sidebar/Sidebar.jsx (hunk 1 above)
Comment (abhijeetekad) [reply src/Components/Sidebar/Sidebar.jsx:11]: Yes

File: src/Components/NewNote/NewNote.css (hunk 2 above)
Comment (abhijeetekad) [reply src/Components/NewNote/NewNote.css:11]: Done

**Pull request merged**